components/expenses/TaxFormLinkModal.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { gql, useMutation } from '@apollo/client';
+import { get } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+
+import { API_V2_CONTEXT } from '../../lib/graphql/helpers';
+
+import Container from '../Container';
+import { Box } from '../Grid';
+import MessageBox from '../MessageBox';
+import MessageBoxGraphqlError from '../MessageBoxGraphqlError';
+import StyledButton from '../StyledButton';
+import StyledInput from '../StyledInput';
+import StyledInputField from '../StyledInputField';
+import StyledLink from '../StyledLink';
+import StyledSelect from '../StyledSelect';
+import { H2, Span } from '../Text';
+import { TOAST_TYPE, useToasts } from '../ToastProvider';
+
+import StyledModal, { ModalBody, ModalHeader } from './../StyledModal';
+
+const setTaxFormMutation = gql`
+  mutation SetTaxForm($account: AccountReferenceInput!, $taxFormLink: URL!, $year: Int!) {
+    setTaxForm(account: $account, taxFormLink: $taxFormLink, year: $year) {
+      success
+    }
+  }
+`;
+
+const arrayOfYears = () => {
+  const max = new Date().getFullYear();
+  const min = 2018;
+  const years = [];
+
+  for (let i = max; i >= min; i--) {
+    years.push(i);
+  }
+  return years;
```

Comment:
:ice_cream: lodash to the rescue

```suggestion
  return range(2018, new Date().getFullYear());
```

Reply:
Thanks for the simplification. 👍🏽